package.json

```diff
@@ -14,6 +14,7 @@
     "@formatjs/intl": "^1.9.7",
     "@giphy/js-components": "^5.0.5",
     "@giphy/js-fetch-api": "^4.0.1",
+    "@panzoom/panzoom": "^4.4.3",
```

Comment:
When adding a new dependency, we need to increase `PROVISION_VERSION` in `version.py` by `0.1`; that will result in folks being prompted to run the provision tool when we first rebase past this commit.

Reply:
When is the appropriate time to do this? In the PR itself? I presume you want it to be the last thing you do in case the version is updated somewhere else beforehand. Or maybe It's done as a separate commit in main?

Reply:
Usually we do it in the commit that adds the new dependency, since then if we merge just that commit, things will work properly. 

But you're right that one can end up with that junk disappearing when rebasing, which is annoying; we have yet to design a fancier `PROVISION_VERSION` system that solves that problem, but do have a few ideas.

Reply:
I rebased the first commit to update `PROVISION_VERSION`.